packages/backend/discovery/kroma/ethereum/meta.json

```diff
@@ -0,0 +1,724 @@
+          "type": null
+        },
+        "PORTAL": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "REQUIRED_BOND_AMOUNT": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "ROUND_DURATION": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "SECURITY_COUNCIL": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "TAX_DENOMINATOR": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "TAX_NUMERATOR": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "TRUSTED_VALIDATOR": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "validatorCount": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "VAULT_REWARD_GAS_LIMIT": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "version": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "validators": {
+          "description": null,
+          "severity": null,
+          "type": null
+        }
+      }
+    },
+    {
+      "name": "Poseidon2",
+      "values": {}
+    },
+    {
+      "name": "Colosseum",
+      "values": {
+        "BISECTION_TIMEOUT": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "CREATION_PERIOD_SECONDS": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "DUMMY_HASH": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "L2_ORACLE": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "L2_ORACLE_SUBMISSION_INTERVAL": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "MAX_TXS": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "PROVING_TIMEOUT": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "SECURITY_COUNCIL": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "version": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "ZK_MERKLE_TRIE": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "ZK_VERIFIER": {
+          "description": null,
+          "severity": null,
+          "type": null
+        },
+        "challengedRoots": {
```

Comment:
Please add a description (maybe even type + severity), since this is what has been updated

Reply:
Added